v22.2/inverted-indexes.md

```diff
@@ -61,19 +61,19 @@ This lets you search based on subcomponents.
 
 You can use GIN indexes to improve the performance of queries using `JSONB` or `ARRAY` columns. You can create them:
 
-- At the same time as the table with the `INVERTED INDEX` clause of [`CREATE TABLE`](create-table.html#create-a-table-with-secondary-and-gin-indexes).
 - For existing tables with [`CREATE INDEX`](create-index.html):
+- At the same time as the table with the `INVERTED INDEX` clause of [`CREATE TABLE`](create-table.html#create-a-table-with-secondary-and-gin-indexes).
```

Comment:
nit: This feels a little abrupt here after `CREATE TABLE`. It might be nice to introduce the two sub-bullets with a couple of words. I assume these are one of two methods one would use depending on their scenario? Only something along the lines of " ... with either of the following relevant options/methods/?"